### PR TITLE
Init version sans license

### PR DESCRIPTION
**Pull request by viduc**
| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

**Comment by viduc** [comment]
Hello,
My bundle depend to symfony/security-guard andi see in the README that our bundle must not depend to symfony/security, but nothing with security-guard, is that the same validation?
Thanks

File: viduc/casbundle/1.0/manifest.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+    "bundles": {
+        "Viduc\\CasBundle\\CasBundle": [
+            "all"
+        ]
+    },
+    "env": {
+        "CAS_HOST": "monserveurcas.fr",
+        "CAS_PORT": "443",
+        "CAS_VERSION": "2."
```
Comment (Nyholm) [inline viduc/casbundle/1.0/manifest.json:10]: This environment variable is never used.

Comment (viduc) [reply viduc/casbundle/1.0/manifest.json]: Thanks for your response.
I delete CAS_VERSION but not CAS_HOST and CAS_VERSION, i use it in configuration (cas.yaml)

**Comment by Nyholm** [comment]
Oh. Im sorry. I've all forgotten about this.

**Comment by Nyholm** [comment]
Could you rebase this so the CI is green. I cannot merge this in the current state.

File: viduc/casbundle/1.0/config/packages/cas.yaml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+cas:
+    host: '%env(CAS_HOST)%'
+    port: '%env(CAS_PORT)'
```
Comment (fabpot) [inline viduc/casbundle/1.0/config/packages/cas.yaml:3]: missing %

**Review by fabpot — COMMENTED** [review]
Closing as there is no more activity and the PR cannot be merged as is. Feel free to reopen if you like.

**Pull request closed without merging**